README.md

```diff
@@ -57,16 +57,15 @@ NFD is used to label nodes with the following labels:
 - RDMA capability
 - GPU features*
 
+>__NOTE__: We use nodeFeatureRules to label PCI vendor and device and it's enabled by default using `nfd.deployNodeFeatureRules` flag
+
 __Example NFD worker configurations:__
 
 ```yaml
     config:
       sources:
         pci:
           deviceClassWhitelist:
-          - "02"
```

Comment:
why do we need to drop these values?

Reply:
we create the labels via nodefeaturerule CR now, so technically we dont need to have them "ON" by default.

I see @ArangoGutierrez prefers to keep as well.